braindecode/models/deepsleepnet.py

```diff
@@ -197,6 +197,7 @@ def __init__(self, n_classes=5, return_feats=False):
         self.fc = nn.Sequential(nn.Linear(3072, 1024, bias=False),
                                 nn.BatchNorm1d(num_features=1024))
 
+        self.features_extractor = nn.Identity()
```

Comment:
@agramfort Do you think I can call an Identity layer feature_extractor in order to use the hook of my project's function to get the feature output ? Maybe put a comment to make it clearer ?
Otherwise the last laye before classifier is a dropout, I don't really know how to deal with it except if I change the name of this only dropout layer.

Reply:
can you add a test using the hook to obtain features? Doing so everyone can see why you need it and if someone remove the identity someday it will break a test.